.tool-versions

```diff
@@ -1,3 +1,4 @@
 scarb 2.7.0
 starknet-foundry 0.18.0
 nodejs v20.11.1
+pnpm 9.7.0
```

Comment:
pnpm version should be managed by corepack

```sh
# once Node.js is installed via asdf or whatever vm,
corepack enable pnpm
```

Reply:
fair

Reply:
can we have it here anyway? I am using asdf for pnpm

Reply:
I prefer to avoid duplication of versioning

https://github.com/cartridge-gg/controller/blob/1bc2065f4cad65f23211959557e618f2215109b6/package.json#L3